include/plat/marvell/a8k/common/plat_config.h

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
would prefer to rename the file to a more specific name line "a8k_common.h" or something.